src/main/java/com/targomo/client/api/request/config/RequestConfigurator.java

```diff
@@ -444,6 +447,10 @@ private static StringBuilder getTargets(final TravelOptions travelOptions) {
         return targetsBuilder;
     }
 
+    private static StringBuilder getTargetGeohashes(final TravelOptions travelOptions) {
```

Comment:
you probably don't need this as extra method. it is private, just used once and just a one liner